controllers/admin/AdminOrderPreferencesController.php

```diff
@@ -79,6 +79,13 @@ public function __construct()
                         'cast' => 'floatval',
                         'type' => 'price'
                     ),
+                    'PS_ORDER_RECALCULATE_SHIPPING' => array(
+                        'title' => $this->trans('Recalculate shipping cost after order edition', array(), 'Admin.ShopParameters.Feature'),
+                        'hint' => $this->trans('Automatically updates the shipping costs when you edit an order.', array(), 'Admin.ShopParameters.Help'),
```

Comment:
So, can you confirm all translations are fine ? I dont want you to kill me :fearful: